Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkController.cs

```diff
@@ -1,264 +1,257 @@
-            public Slider x3Slider;
-            public Slider threshold0Slider;
-            public Slider threshold1Slider;
-            public Slider threshold2Slider;
-            public Slider threshold3Slider;
+        private float _initGain;
+        private float _initTreble;
+        private float _initBass;
+        private float _initFadeLength;
+        private float _initFadeExpFalloff;
+        private float _initX0;
+        private float _initX1;
+        private float _initX2;
+        private float _initX3;
+        private float _initThreshold0;
+        private float _initThreshold1;
+        private float _initThreshold2;
+        private float _initThreshold3;
 
-            private float _initGain;
-            private float _initTreble;
-            private float _initBass;
-            private float _initFadeLength;
-            private float _initFadeExpFalloff;
-            private float _initX0;
-            private float _initX1;
-            private float _initX2;
-            private float _initX3;
-            private float _initThreshold0;
-            private float _initThreshold1;
-            private float _initThreshold2;
-            private float _initThreshold3;
+        private RectTransform _threshold0Rect;
+        private RectTransform _threshold1Rect;
+        private RectTransform _threshold2Rect;
+        private RectTransform _threshold3Rect;
 
-            private RectTransform _threshold0Rect;
-            private RectTransform _threshold1Rect;
-            private RectTransform _threshold2Rect;
-            private RectTransform _threshold3Rect;
+        #region PropertyIDs
 
-            #region PropertyIDs
-            
-            // ReSharper disable InconsistentNaming
-            private int _X0;
-            private int _X1;
-            private int _X2;
-            private int _X3;
-            private int _Threshold0;
-            private int _Threshold1;
-            private int _Threshold2;
-            private int _Threshold3;
-            // ReSharper restore InconsistentNaming
+        // ReSharper disable InconsistentNaming
+        private int _X0;
+        private int _X1;
+        private int _X2;
+        private int _X3;
+        private int _Threshold0;
+        private int _Threshold1;
+        private int _Threshold2;
+        private int _Threshold3;
+        // ReSharper restore InconsistentNaming
 
-            private void InitIDs()
-            {
-                _X0 = PropertyToID("_X0");
-                _X1 = PropertyToID("_X1");
-                _X2 = PropertyToID("_X2");
-                _X3 = PropertyToID("_X3");
-                _Threshold0 = PropertyToID("_Threshold0");
-                _Threshold1 = PropertyToID("_Threshold1");
-                _Threshold2 = PropertyToID("_Threshold2");
-                _Threshold3 = PropertyToID("_Threshold3");
-            }
+        private void InitIDs()
+        {
+            _X0 = VRCShader.PropertyToID("_X0");
```

Comment:
doesn't this break standalone?

Reply:
oh, the whole script is `#if UDONSHARP` now ?

Reply:
I'm fine with keeping it multiplatform but you previously said that the controller is currently only for Udon, looking through the code I don't see anything vrc specific